### PR TITLE
Push the complete apple-clone-website to master

### PR DESCRIPTION
**Pull request by harshdeepkanhai**
Building with Backgrounds and Gradients
-----------------------------------------------------------

- Add background for hero and promo sections
- Add linear gradient for the navbar

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @harshdeepkanhai, great job on this amazing project :+1: README is professional, Commit messages descriptive and the project looks great. However, we can improve our project and get our project approved by addressing the comments below: :slightly_smiling_face:
- We currently have a huge space at the footer, let us fix it
xtra
- Let us fix the margin between the hero image and the four buttons
mar
- Let us add a link to the copyright as required
mar1
- There are some repeating codes in our CSS file, let us apply the DRY rule
- The HTML validator returned 1 warning which is easily fixable

Kindly fix these comments and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.flex-2 {
+  display: flex;
+  justify-content: center;
+}
```
Comment (andresporras3423) [inline assets/stylesheets/main.css]: group selectors:
```suggestion
.flex, .flex-2 {
  display: flex;
  align-items: center;
}
```
or create a new class:
```suggestion
.new_class_name {
  display: flex;
  align-items: center;
}
```

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @harshdeepkanhai** 
Good job with your new submission for this project :muscle:  
Your project passes most of the requirements.  :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.
- [x]  Check the footer of your page, it looks like this on my screen (1321px wide):



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far @harshdeepkanhai 👍 
## However, there are some changes required 👀 🧰 🔨 🔧 

- On my screen, the navbar is not centered, and the footer text is misplaced:

image

image

Your page is supposed to look good for all screen sizes. To test this, you may change the zoom % on your browser, to check that the positioning does not become scrambled when you change the screen size.

You may use relative width units like `vw` or `%` to make your page layout more [fluid](https://alistapart.com/article/fluidgrids/).

### Submit these changes and request a new code review! 😄 👍 
### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by bertil291utn** [comment]
# Status: on review 🟠 

##### *Reach out to me on* [Twitter](bit.ly/bertil-twitter) , [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community as @btandayamo

**Comment by bertil291utn** [comment]
# Project Approved ✔️ 

Hi, @harshdeepkanhai  your project was approved, please merge to master branch and move on with the next assignments.
You're doing  a great job keep going 💻 🚀 🔥


[OPTIONAL]
In order to approach a better project which looks like exactly clon as the actual apple web page 
- Be careful with the typos 
- Responsiveness page
   Add a min-width property inside the body attribute and try to use % instead of px

##### *Reach out to me on* [Twitter](bit.ly/bertil-twitter), [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community like @btandayamo

**Pull request merged**